plugins.txt

```diff
@@ -37,7 +37,15 @@ credentials-binding:523.vd859a_4b_122e6
 dark-theme:185.v276b_5a_8966a_e
 datadog:4.0.0
 design-library:104.v492ed3cce3f2
-ec2:1.68
+ec2:ec2-1.69-rc1489.404b_49d31b_e7
+aws-java-sdk-ec2:1.12.163-315.v2b_716ec8e4df
+apache-httpcomponents-client-4-api:4.5.13-1.0
+aws-credentials:189.v3551d5642995
+bouncycastle-api:2.25
+node-iterator-api:1.5.1
+trilead-api:1.0.13
+javax-activation-api:1.2.0-2
+javax-mail-api:1.6.2-5
```

Comment:
Explicit missing dependencies needed for building with the incremental version of the ec2-pliugin